packages/manager/modules/models/support/ticket/ticket.class.js

```diff
@@ -1,10 +1,15 @@
 import capitalize from 'lodash/capitalize';
+import { STATUS } from './ticket.constants';
 
-export default class SupportTicket {
+export default class {
   constructor(ticket) {
     Object.assign(this, ticket);
   }
 
+  static get Status() {
+    return { ...STATUS };
```

Comment:
I don't get what this is supposed to do

Reply:
maybe not, but some people like having a lots of files ^^